frontend-react/src/components/Admin/AdminReceiverDashboard.test.tsx

```diff
@@ -81,19 +85,21 @@ const mockData: AdmConnStatusDataType[] = [
 ];
 // </editor-fold>
```

Comment:
We can remove the `editor-fold` stuff since there's no reason to change code for specific IDEs.  Not a huge deal, we can do that cleanup anytime!